### DISCUSSION TITLE
Exclude field from saving

### DISCUSSION THREAD

**Discussion by ahmetkocabiyik** [filamentphp/filament#9808] [Help] [answered]
I used Model::unguard in AppServiceProvider boot method. So in our model we didn't use any fillable array. I want to exclude one field from saving but don't want to disable. Because we used this field to fill a text input with some value when click. Is there any method to exclude this field from saving.

**Comment by leandrocfe** [comment, marked as answer]
`->dehydrated(false)`

**Reply by ahmetkocabiyik** [reply]
Thank you so much

**Reply by wali-eldin-hassan** [reply]
does it work with Wizard\Step ??
